amplify-tools-gradle-plugin/build.gradle

```diff
@@ -0,0 +1,25 @@
+apply plugin: 'groovy'
+apply plugin: 'idea'
+apply plugin: 'maven-publish'
+
+apply plugin: 'com.android.library'
+
+sourceSets.main.groovy.srcDirs = ["src/main/java", "src/main/groovy"]
+
+dependencies {
+    implementation 'androidx.appcompat:appcompat:1.1.0'
```

Comment:
We define all of our dependencies in the project `build.gradle`, and then reference them as `implementation dependency.androidx.appcompat`. This ensures that we have a mutually consistent set of dependencies across the project modules.